src/models/commons.rs

```diff
@@ -31,6 +34,17 @@ pub enum ParamValue {
     ArrayValue(Vec<ParamValue>),
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, Hash, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct ValueDecoded {
+    pub operation: String, //TODO use operation, usually this is received as 0 or 1 but here it is `CALL`, etc. as a string
```

Comment:
Do we want to create an issue somewhere for this? ... not sure if we want to keep this TODO in here